### ISSUE TITLE
Add visualization for comparing equity and equality

### ISSUE THREAD

**Issue by konrad** [carpentries/instructor-training#1432]
There is a great visualization that helps to show the difference between equity and equality: 

![](https://live.staticflickr.com/7558/27755848262_2e21d97151_w_d.jpg)

(source: https://www.flickr.com/photos/communityeyehealth/27755848262)

There exist several derivatives of this image and every time I am teaching the instructor training somebody is pointing to one of them. :) I think it would make to include one of them to make the abstract difference easier to grasp. The images linked aboce seems to be under a CC-BY-NC license so reuse would be possible.

**Comment by brownsarahm** [comment]
We got a recent PR of this or similar too #1393 but we need to be sure that it's truly licensed I think and not someone who inappropriately took it and put it somewhere with the license.  For example, [this source suggests they paid someone to make it CC-BY-SA](https://interactioninstitute.org/illustrating-equality-vs-equity/). 

Also, I'm not sure about he CC-BY-NC (or SA) being enough given that the curricula are CC-BY. My (non expert) understanding is that we cannot, becuase -NC and -SA are less permissive. 

The [cdh says that datasets have to be CC0](https://carpentries.github.io/curriculum-development/designing-challenges.html) but maybe that does not apply to images?

**Comment by fiveop** [comment]
Yes, see [this chart](https://wiki.creativecommons.org/wiki/Wiki/cc_license_compatibility) on the Creative Commons Wiki and the key phrase “Use at least the most restrictive licensing of the two (use the license most to right or down state) for the new work.”

**Comment by brownsarahm** [comment]
So, we have a final conclusion on this image not being included, and now we need to decide how &where  to document this for future people who want to contribute this image, because with it twice in the past few months, I expect it to happen again, since it is a fitting image. 

@carpentries/instructor-training-maintainers what do you all think?

**Comment by tobyhodges** [comment]
It feels like something for the CONTRIBUTING.md to me. If so, we should add it to our template for that file, so that it is included in all lesson repositories in future. I also need to make sure that this gets mentioned in the Lesson Development Training curriculum...

**Comment by ndporter** [comment]
Agreed, @tobyhodges that CONTRIBUTING.md is a good place, probably in the "what not to contribute" we could add something about not adding images/content from other sources unless it's CC0/public domain or CC-BY. It might also be good to add notes about limiting external links while we're at it.
